en/docs/report-vulnerabilities.md

```diff
@@ -0,0 +1,54 @@
+# Report a Security Issue
+
+We welcome all contributions from our user community, developers, customers and security researchers to reinforce our 
+security. We strongly encourage you to report security issues to our private and highly confidential security mailing 
```

Comment:
```suggestion
security of our products and services. We strongly encourage you to report security issues to our private and highly confidential security mailing 
```

Reply:
Fixed in f075965fb133a67c6e21eabaa22a9225677ce6d8